src/MSBuild/XMake.cs

```diff
@@ -782,6 +791,18 @@ private static void DebuggerLaunchCheck()
                     {
                         ReplayBinaryLog(projectFile, loggers, distributedLoggerRecords, cpuCount);
                     }
+                    else if ((getProperty.Length > 0 || getItem.Length > 0 || getTargetResult.Length > 0) && FileUtilities.IsSolutionFilename(projectFile))
```

Comment:
nit: `getProperty.Length > 0 || getItem.Length > 0 || getTargetResult.Length > 0` is repeated 3 times in this method. Would you consider adding a local variable with this value and a good name?

Reply:
I'm not a huge fan of the name I came up with, since it's rather verbose, but I liked other options less. If you can think of something better, please opine 🙂